CMakeLists.txt

```diff
@@ -196,7 +196,6 @@ add_to_dist(
     COPYING
     COPYING-DOCS
     ChangeLog
-    HACKING
```

Comment:
This file is still mentioned in README.

Tip: Before removing files run a command like `grep -inr 'HACKING'` in the top source directory.

Reply:
Thanks for the heads up @fellen . I assume you will handle this trivial fix yourself ?